api/proto/openapiv1/dop/dop.proto

```diff
@@ -409,6 +409,7 @@ message CMDB_CERTIFICATE_GET_Request {
 }
 message CMDB_ITERATION_DELETE_Request {
     string id = 1; // generated from path variable: id. You should change the proto type if necessary.
+    bool  onlyIteration = 2;
```

Comment:
format it

Reply:
protos under `openapiv1` dir is auto generated, so don't modify it manually.

Reply:
ok，modified